index.bs

```diff
@@ -120,6 +121,12 @@ A [=crash report=]'s [=report/body=], represented in JavaScript by
   "true" or "false", representing whether the {{Document}} that crashed belonged
   to a [=/top-level traversable=], or not.
 
+* <dfn for="CrashReportBody" noexport>page_visibility</dfn>: The string value
+  "<code>visible</code>" or "<code>hidden</code>" derived from the {{DocumentVisibilityState}}
```

Comment:
How is it derived from that enum? Should we explicitly say "`true` if {{DocumentVisibilityState}} is **visible**; false otherwise"?

Reply:
Hmm, well it's not implemented as "true" or "false". It is derived from the enum value as the description states, it is either "visible" or "hidden" to match the enum. I felt that was more descriptive, and our partners were pretty happy with that. Is that OK with you? The latest commit removes "derived" to make the wording more clear.

Reply:
Ahh, that makes sense; I assumed it was still a bool because I didn't read it carefully enough 😁